.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```

Comment:
Several files have been added to the `.idea`  folder, including this file. Are these all necessary files? If not, please remove them.